sdk/storage/azure-storage-files-datalake/src/datalake_file_system_client.cpp

```diff
@@ -3,7 +3,9 @@
 
 #include "azure/storage/files/datalake/datalake_file_system_client.hpp"
 
+#include <azure/core/azure_assert.hpp>
 #include <azure/core/http/policies/policy.hpp>
+
```

Comment:
extra new line

Reply:
mm, don't we want to keep core and storage headers separated by one space?  I though that was helpful.  But I can remove it. It won't matter at all at the end LOL

Reply:
Is there any reason why we should consider following what storage is doing for including headers as a guideline?

I am curious if it was just an arbitrary choose to do it like that or if it is making sure about something during compilation.

Doxygen format do the alphabet ordering, so there would be an empty line between each number in your list.

Reply:
> Is there any reason why we should consider following what storage is doing for including headers as a guideline?

No, I'm not asking that. There's no guideline for this. I take it to mean, storage team can pick our own way. If you make any changes to storage code, it'd be better to keep consistent with existing code.

If core sdk team can come up with some guideline, we'd be happy to follow it.

Reply:
> I am curious if it was just an arbitrary choose to do it like that or if it is making sure about something during compilation.

https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

Reply:
> Doxygen format do the alphabet ordering, so there would be an empty line between each number in your list.

Yeah, there are five groups in total. There's a new line between them. But there's no new line within each group.